app/frontend/src/features/donations/Donations.tsx

```diff
@@ -0,0 +1,117 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import Markdown from "components/Markdown";
+import {
+  DONATIONS_BANNER_TEXT,
+  DONATIONS_BANNER_TITLE,
+  DONATIONS_TEXT,
+  DONATIONS_TEXT2,
+  DONATIONS_TITLE,
+} from "features/donations/constants";
+import Landscape from "features/donations/resources/landscape.jpeg";
+import CouchersLogo from "resources/CouchersLogo";
+
+import DonationsBox from "./DonationsBox";
+
+const useStyles = makeStyles((theme) => ({
+  donationsImage: {
+    width: "100%",
+    height: "100%",
+    objectFit: "cover",
+    opacity: 0.3,
+  },
+
+  donationsLogo: {
+    "&&": {
+      height: "72px",
+      width: "initial",
+    },
+  },
+
+  donationsWrapper: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    [theme.breakpoints.down("sm")]: {
+      height: "8rem",
+    },
+  },
+
+  donationsLogoHeader: {
+    position: "absolute",
+    zIndex: 1,
+    maxWidth: "1100px",
+    display: "flex",
+    alignItems: "center",
+    width: "100%",
+    [theme.breakpoints.down("md")]: {
+      maxWidth: "672px",
+    },
+    [theme.breakpoints.down("sm")]: {
+      margin: theme.spacing(0, 3),
+    },
+  },
+
+  donationsPage: {
+    display: "grid",
+    gridTemplateColumns: "624px 408px",
```

Comment:
These seems like highly specific "magic numbers" - could this not be done with `fr` units instead?

Reply:
624px equals to 27 columns of 24px and 408px to 17 columns of 24px. The donations box width was set by the design mockup. The width of the text was adjusted by me in order to have an optimal text length (between 65 to 80 characters per line) taking in account the font size (16px).